src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -489,6 +489,25 @@ def _get_extension_suppressions(mod_loaders):
 
         return self.command_table
 
+    def sort_command_loaders(self, command_loaders):
+        core_command_loaders = []
```

Comment:
We usually call them modules:

```suggestion
        module_command_loaders = []
```

`core` has a special meaning as `azure-cli-core`.

Reply:
Yes, they are in one repo, but `modules` usually refer to `azure.cli.command_modules` (path in repo: `src/azure-cli/azure/cli/command_modules`) while `core` refers to `azure.cli.core` (path in repo: `src/azure-cli-core/azure/cli/core`). They are different packages:

https://github.com/Azure/azure-cli/blob/dc4393dfe5319df8ce4190f12f3f981ff8f49563/src/azure-cli/setup.py#L161

https://github.com/Azure/azure-cli/blob/3a2c33aa525d4718c9c277e2aa3ce5c8c4d50a20/src/azure-cli-core/setup.py#L73